README.md

```diff
@@ -36,27 +37,61 @@ In particular, you can provide them with `--release`, `--dev`, or `--profile`
 flags to switch the compilation profile. If not option is provided, the scripts
 default to the `--release` profile.
 
-### Running application and examples
 For best experience, it is recommended to use the 
-`scripts/watch.sh --dev` in a second shell. In order to build the IDE application, 
-follow the steps below:
+`scripts/watch.sh --dev` in a second shell.
+
+### Web Application
+In order to build the IDE web application, follow the steps bellow:
 
 ```bash
-cd app
-npm install
-npm run start 
+ide/app$ npm install
+ide/app$ npm run web:dev 
 ```
 
 You can now navigate to http://localhost:8080 and play with it! The example
-scenes will be available at http://localhost:8080/debug.
+scenes will be available at http://localhost:8080/?debug.
 
 While Webpack provides handy utilities for development, like live-reloading on
 sources change, it also adds some runtime overhead. In order to run the compiled
 examples using a lightweight http-server (without live-reloading functionality),
-please use the `npm run prod-server` command.
+please use the 
+```
+ide/app$ npm run web:prod
+```
 
 **Please remember to disable the cache in your browser during development!**
 
+### Desktop Application
+In order to build the IDE desktop application, follow the steps bellow:
+
+```bash
+ide/app$ npm install
+ide/app$ npm run electron:dev
+```
+
+To debug a subsystem, use the `--debug` argument.
+
+```bash
+ide/app$ npm run electron:dev -- --debug        # Shows the selection screen.
+ide/app$ npm run electron:dev -- --debug=shapes # Runs shapes' subsystem.
+```
+
+If you want to distribute the desktop application, run the following steps instead:
+
+```bash
+ide/app$ npm install
+ide/app$ npm run electron:dist
+```
+
+The generated executable will be available at `dist/enso-ide-major.minor.*`. Its extension will
```

Comment:
to follow unix convention, all variables in names should bre capitalized: `dist/enso-ide-MAJOR.MINOR.*`.